### PR TITLE
Structure lists properly by removing invalid elements under ul element

### PR DESCRIPTION
**Pull request by gathogojr**
Structure lists properly by removing invalid elements under ul element

### CODE REVIEW

**Review by marabooy — APPROVED** [review]

File: _posts/2014-07-09-trippin-new-odata-v4-sample-service.html [hunk 1]
```diff
@@ -25,33 +25,47 @@ <h2>Model</h2>
 <h2>Feature Coverage</h2>
 <p>As we mentioned in the introduction part, TripPin tries to cover as many features as possible. Below are the main features we covered in TripPin.</p>
 <ul>
-<li><b>Basic entity /  complex type and their collections</b></li>
+<li><b>Basic entity /  complex type and their collections</b>
 <p>For instance, <code>Person</code> is an entity and <code>Country</code> is a complex type. <code>People</code> is a collection of entity and <code>AddressInfo</code> is a collection of complex type.</p>
-<li><b>Navigation property</b></li>
+</li>
```
Comment (habbes) [inline _posts/2014-07-09-trippin-new-odata-v4-sample-service.html:30]: Some indentation would make it easier to tell the hierarchy of these elements and what owns what.

**Review by habbes — APPROVED** [review]
Looks good, but hard to review do to lack of indentation in the markup code.

File: _posts/2014-07-09-trippin-new-odata-v4-sample-service.html (hunk 1 above)
Comment (gathogojr) [reply _posts/2014-07-09-trippin-new-odata-v4-sample-service.html:30]: @habbes I did consider that but I was afraid it'd be hard to pick out the changes relevant to the reported issue. Formatting would affect the whole page - not to mention that nearly the entire site currently has markup formatting issues

Comment (wandeg) [reply _posts/2014-07-09-trippin-new-odata-v4-sample-service.html:30]: We should actually do that for the entire site

**Review by wandeg — APPROVED** [review]

**Pull request merged**